benchmark/GasCostBenchmark.js

```diff
@@ -0,0 +1,298 @@
+const DocumentStore = artifacts.require("./DocumentStore.sol");
```

Comment:
Some overall feedbacks:
1. I think in the case benchmark would be nice once the instance of the contract has been, created to provide it to a function running all the actions to perform on the contract: `runBenchmark(contract)` => this will ensure the contract perform the same steps (if I'm not wrong only contract creation is different)

2. Would be easier to read the result of the benchmark it it was like the following:
```
            contractNameA     contractNameB
stepName1          2                4
stepName2          8                16
```
this can be achieve with the following structure:
```
console.table({
stepName1: {contractNameA: 2, contractNameB: 4},
stepName2: {contractNameA: 8, contractNameB: 16},
})
```

3. Would it be nice to keep a record of your benchmark in the readme and to describe on which conditions it has been run ?

Reply:
run it in CI? :D

Reply:
Laurent: Done.
Rj: Another PR maybe. Not sure if it's useful in CI and that's not the intention of this PR.